src/main/java/com/uc/BoredNoMore/controllers/MainController.java

```diff
@@ -38,10 +47,22 @@ public String getActivities() {
 	 * @return The template for adding custom activities
 	 */
 	@GetMapping("/addactivity")
-	public String getAddActivity() {
+	public String getAddActivity(Model model) {
+		model.addAttribute("activity", new ActivityDTO());
 		return "addactivity";
 	}
 	
+	/**
+	 * Post the activity to the database and return to the activities page
+	 * @return void
+	 */
+	@PostMapping("/addactivity")
+	public String postAddActivity(@ModelAttribute ActivityDTO activity, RedirectAttributes redirectAttributes) {
+		//TODO: Instantiate IActivityDAO and call ActivityDAO.addActivity(activity); to save to database
```

Comment:
In regards to this line: Since we do not have a database yet, we have yet to make a service to instantiate our interface to save to the database. I added this as a TODO that will get updated upon the completion of the stack,